From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: a821 <a821@nospam.mail.de>
Date: Wed, 21 Oct 2026 20:28:24 +0200
Subject: [PATCH] fix many deprecation errors

* remove deprecated `bdist_wheel`
* remove LICENSE classifiers
* remove pinned build dependencies (unneeded for arch)
---
 pyproject.toml | 5 ++---
 setup.cfg      | 5 -----
 2 files changed, 2 insertions(+), 8 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 9e5511c..ca479fc 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,8 +1,7 @@
 [build-system]
 requires = [
-    "setuptools; python_version != '3.3'",
-    "setuptools<40.0; python_version == '3.3'",
-    "setuptools_scm<8.0"
+    "setuptools",
+    "setuptools_scm"
 ]
 build-backend = "setuptools.build_meta"
 
diff --git a/setup.cfg b/setup.cfg
index a9d9bcb..aa09405 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,3 @@
-[bdist_wheel]
-python_tag = py2.py3
-
 [metadata]
 name = python-dateutil
 description = Extensions to the standard Python datetime module
@@ -18,8 +15,6 @@ license_file = LICENSE
 classifiers =
     Development Status :: 5 - Production/Stable
     Intended Audience :: Developers
-    License :: OSI Approved :: BSD License
-    License :: OSI Approved :: Apache Software License
     Programming Language :: Python
     Programming Language :: Python :: 2
     Programming Language :: Python :: 2.7
-- 
2.56.0

